### PR TITLE
Added Mass Insertions to QueryBuilder

### PR DESCRIPTION
**Pull request by calebkleveter**
Adds mass insertions to `QueryBuilder`, so you can save an array of models with a single database operation:

```swift
let users = [User(name: "Greg"), User(name: "Caleb"), User(name: "Tanner")]
users.create(on: connection)
```

Fixes https://github.com/vapor/fluent/issues/569

Requires: https://github.com/vapor/fluent-sqlite/pull/26, https://github.com/vapor/fluent-postgresql/pull/99, https://github.com/vapor/fluent-mysql/pull/129

### CODE REVIEW

File: Sources/FluentSQL/FluentSQLQuery.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ public protocol FluentSQLQuery {
     var groupBy: [GroupBy] { get set }
     var limit: Int? { get set }
     var offset: Int? { get set }
-    var values: [String: Expression] { get set }
+    var values: [[String: Expression]] { get set }
```
Comment (tanner0101) [inline Sources/FluentSQL/FluentSQLQuery.swift]: This is a public protocol. So we unfortunately can't change it unless we do a major version bump.

File: Sources/Fluent/Model/Model+CRUD.swift [hunk 2]
```diff
@@ -95,3 +95,11 @@ extension Future where T: Model {
         }
     }
 }
+
+extension Array where Element: Model {
+    
+    /// See `Model`.
+    public func save(on connectable: DatabaseConnectable) -> Future<[Element]> {
+        return Element.query(on: connectable).create(self)
```
Comment (tanner0101) [inline Sources/Fluent/Model/Model+CRUD.swift]: Why not just call save multiple times and map it here? I'm guessing you want it to run as a single query?

Comment (calebkleveter) [reply Sources/Fluent/Model/Model+CRUD.swift]: Correct. It's to reduce the number of operations.

**Comment by tanner0101** [comment]
This now exists on `FluentKit` master: https://github.com/vapor/fluent-kit/blob/master/Sources/FluentKit/Model/FluentModel.swift#L47-L65

**Pull request closed without merging**